### DISCUSSION TITLE
minimal x-ray occurrences dosen't seem to work

### DISCUSSION THREAD

**Discussion by zyzy5** [xxyzz/WordDumb#258] [Q&A] [answered]
there are two many x-rays that not necessary were build.
 <img width="376" alt="屏幕截图 2024-09-05 225522" src="https://github.com/user-attachments/assets/bebe5b75-93d4-4c3c-905f-624eb87ab7c8">
![photo_2024-09-05_23-02-07](https://github.com/user-attachments/assets/3847a402-6933-4e5b-8f8f-90fde85bdfa5)
![photo_2024-09-05_23-02-03](https://github.com/user-attachments/assets/118395dc-f7ed-43c2-b499-7f496a4f0a55)

**Comment by zyzy5** [comment]
how do I set the config make it work?

**Reply by xxyzz** [reply, marked as answer]
Non-person entities that have Wikipedia page will not be removed even if they only occurrence once.

I have pushed a commit to remove person entities when the "fetch people" option is disabled: https://github.com/xxyzz/WordDumb/commit/2ea201cd58330238cd16988e48bd945bbd04ceb9

You can download the test version from here: https://github.com/xxyzz/WordDumb/actions/runs/10738545781
